core/windows_and_tabs/window_management_mac.talon

```diff
@@ -0,0 +1,3 @@
+os: mac
+-
+focus last: key(cmd-tab)
```

Comment:
I'd be inclined to implement this functionality via an action so it's easier to maintain a consistent command set across operating systems. The downside of maintaining multiple .talon in this fashion is that it makes it a little easier for the commands to drift apart. If implemented via an action, we can maintain a window_management file and override the actions as necessary depending on the operating system.

Reply:
I started there but struggled with understanding the existing windows and application management python files. Do you think you could do a PR to demonstrate how that would need to change? I'll take another look when I can. I really appreciate the feedback!

Reply:
I took a stab at this. I don't have a Linux machine handy I took a guess at how to implements the same functionality. I would love a second set of eyes if someone has a chance.